checks/evaluation/contributors_test.go

```diff
@@ -20,134 +20,75 @@ import (
 	"github.com/google/go-cmp/cmp/cmpopts"
 
 	"github.com/ossf/scorecard/v4/checker"
-	"github.com/ossf/scorecard/v4/clients"
-	"github.com/ossf/scorecard/v4/utests"
+	"github.com/ossf/scorecard/v4/finding"
 )
 
 func TestContributors(t *testing.T) {
 	t.Parallel()
-	testCases := []struct {
+	type args struct { //nolint
 		name     string
-		raw      *checker.ContributorsData
-		expected checker.CheckResult
+		findings []finding.Finding
+	}
+	tests := []struct {
```

Comment:
let's clean up these definitions like in https://github.com/ossf/scorecard/pull/3409:
1. get rid of args
2. use a `scut.TestReturn` instead of `checker.CheckResult`
3. use ValidateTestReturn for validation

Reply:
Should all have been covered